charts/agent-control/templates/rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if (index .Values "agent-control-deployment").enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
```

Comment:
aren't you missing:
```
    helm.sh/hook: post-install,post-upgrade
    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
```

Reply:
True. I added it to the role, the role binding and the service account. They are all used only for the installation job.